privateca/snippets/test_subordinate_ca.py

```diff
@@ -41,11 +41,17 @@ def generate_name() -> str:
 # We are hitting 5 CAs per minute limit which can't be changed
 # We set the backoff function to use 4 as base - this way the 3rd try
 # should wait for 64 seconds and avoid per minute quota
+# Adding some random amount of time to the backoff timer, so that we
+# don't try to call the API at the same time in different tests running
+# simultaneously.
 def backoff_expo_wrapper():
-    return backoff.expo(base=4)
+    for exp in backoff.expo(base=4):
+        if exp is None:
+            yield None
+        yield exp * (1 + random.random())
 
 
-@backoff.on_exception(backoff_expo_wrapper, Exception, max_tries=3)
+@backoff.on_exception(backoff_expo_wrapper, Exception, max_tries=4)
```

Comment:
If my calculations are correct the 4th try will have a timeout of 256+ seconds. I think we should stay with 3 tries (64s * 1 + random()).